seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -442,7 +518,7 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_f
     "active_projects": [
         "{guid}"
     ],
-    "vcf_path": "gs://test_bucket/test_path.vcf.gz",
+    "vcf_path": "gs://test_bucket/test_path.vcf",
```

Comment:
Is there any reason to remove `.gz`?

Reply:
yes, it was weird that we use different success request bodies and it made it hard to read/ maintain. This changes it so the successful requests use the same success body, not the body that was previously used to test a failure case